pkg/build/executor.go

```diff
@@ -21,24 +21,35 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (bc *Context) Execute(name string, arg ...string) error {
-	logname := name
-
-	if bc.UseProot {
-		arg = append([]string{"-0", name}, arg...)
-		name = "proot"
-	}
-
-	cmd := exec.Command(name, arg...)
+func runCommand(cmd *exec.Cmd, logname string) error {
 	log.Printf("running: %v", cmd.String())
 
 	output, err := cmd.CombinedOutput()
 	if output != nil {
 		log.Printf("[%s] %s", logname, output)
 	}
 	if err != nil {
-		return errors.Wrapf(err, "failed to run %s", name)
+		return errors.Wrapf(err, "failed to run %s", cmd.String())
```

Comment:
```suggestion
		return errors.Wrapf(err, "failed to run %s", cmd)
```
If underlying type struct implements `String()` function, you can omit using it when passing variable to fmt when you specify `%s`, but you can ignore this suggestion as it's purely style